Ginger/Ginger/App.xaml.cs

```diff
@@ -770,7 +770,7 @@ public static bool SetSolution(string SolutionFolder)
 
                 //Cleanup
                 SolutionCleanup();
-
+                // !!!!!!!!!!!!! '\' is not good
                 if (!SolutionFolder.EndsWith(@"\")) SolutionFolder += @"\";
```

Comment:
i think we can remove this "\" adding If, because we now using Path.Combine and that will add missing "\" if needed